xarray/tests/test_missing.py

```diff
@@ -23,7 +23,17 @@
     requires_dask,
     requires_scipy,
 )
-from xarray.tests.test_cftime_offsets import _CFTIME_CALENDARS
```

Comment:
should be conditionally import instead?

```
try:
	import cftime
	from ... import _CFTIME_CALENDARS
except ImportError:
	pass
```

Reply:
It would be something like this then:

try:
	import cftime
	from ... import _CFTIME_CALENDARS
except ImportError:
	_CFTIME_CALENDARS = []